src-tauri/src/northstar/install.rs

```diff
@@ -168,6 +168,22 @@ pub fn find_game_install_location() -> Result<GameInstall, String> {
     // Attempt parsing Steam library directly
     match steamlocate::SteamDir::locate() {
         Some(mut steamdir) => {
+            #[cfg(target_os = "linux")]
+            {
+                let snap_dir = match std::env::var("SNAP_USER_DATA") {
+                    Ok(snap_dir) => std::path::PathBuf::from(snap_dir),
+                    Err(_) => match dirs::home_dir() {
+                        Some(path) => path,
+                        None => std::path::PathBuf::new(),
+                    }
+                    .join("snap"),
+                };
+
+                if steamdir.path.starts_with(snap_dir) {
+                    log::warn!("Found Steam installed via Snap, you may encounter issues");
```

Comment:
Just to be clear that this warning is not shown to end-user unless they run from CLI. I guess that's fine?

At least I don't know an easy way to propagate message back to UI. From the top of my head if we just return an `Error` it gets caugh somewhere higher up in the stack and just converted to a generic "Couldn't find install".

Might need to rework the error system to use custom error types but that's a whole other beast ^^"

Reply:
I think returning an Error is a bit much.
This isn't a fatal error but known to cause problems.

Maybe the the dialogs from #418 would be good for this?
Open a dialog, ask if the user wants to launch via snap regardless of any issues and continue

Reply:
Yeah, could be an idea. I'd say we me merge as is, and then add dialog here once the other PR is merged.

Reply:
Sounds good